Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM golang:1.23
 
 ARG PROTOC_GEN_GO_VERSION="v1.36.3"
 ARG PROTOC_GEN_GO_GRPC_VERSION="v1.5.1"
+ARG PROTOC_GEN_GO_VTPROTO_VERSION="v0.6.0"
```

Comment:
Please use latest commit. This is quite old and there are some worth commits on main.